tests/system/hw_math/source/main.c

```diff
@@ -7,6 +7,82 @@
 
 #include <nds.h>
 
+#define TEST_SIZE (1000)
+
+u32 dkp_sqrt32(int a)
```

Comment:
Please avoid putting copy-pasted benchmarks for other projects in BlocksDS. Not only can they change the code at any time, which would make the copy paste outdated, it feels in bad taste.

This can be a separate project.

Reply:
Yeah, I agree with asie.

Reply:
I'll remove them then. I could put in a software implementation test instead and rename the output.

Reply:
Yeah, measuring against a simple C-based implementation sounds like a better idea and provides meaningful information to developers; it can also be used to validate correct outputs from the BlocksDS hardware accelerated implementation.